package.json

```diff
@@ -7,7 +7,11 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "i18next": "^19.6.3",
+    "i18next-browser-languagedetector": "^5.0.0",
+    "i18next-http-backend": "^1.0.17",
```

Comment:
are you using these two dependencies? if not so please remove them.

Reply:
i didn't add them really they are from master

Reply:
no if they are from master they will not show here as Plus
any way just remove them

Reply:
and  now you have a conflict just fix it please and I will merge